cocos/2d/assembler/label/bmfontUtils.ts

```diff
@@ -622,10 +647,31 @@ export const bmfontUtils = {
                 this.appendQuad(_comp, texture, _tmpRect, isRotated, letterPositionX - appX, py - appY, _bmfontScale);
             }
         }
-
+        const indexCount = renderData.indexCount;
+        this.createQuadIndices(indexCount);
+        renderData.chunk.setIndexBuffer(QUAD_INDICES);
         return ret;
     },
 
+    createQuadIndices (indexCount) {
+        if (indexCount % 6 !== 0) {
+            console.error('illegal index count!');
+            return;
+        }
+        const quadCount = indexCount / 6;
+        QUAD_INDICES = null;
+        QUAD_INDICES = new Uint16Array(indexCount);
```

Comment:
Is QUAD_INDICES  needed? Could createQuadIndices function returns the  Uint16Array directly?

Reply:
the moments of creating quad indices of various assembler are different

Reply:
This variable is shared by many instances, the access order may cause the hard-to-track data error. I can not see any benefit here.